kpatch-build/create-diff-object.c

```diff
@@ -69,6 +69,8 @@
 		printf(format, ##__VA_ARGS__); \
 })
 
+extern char *find_exported_symbol_objname(const char *fname, char *symname);
```

Comment:
Ah, ok. Then I'll simply move the parse-mod-symvers functions to lookup.c, rename the function to lookup_exported_symbol_objname(), and add in the copyrights. Thanks for the clarifications.